source/test/project.xml

```diff
@@ -13,6 +13,7 @@
 	
 	<haxelib name="munit" />
 	<haxelib name="flixel" />
+	<haxelib name="flixel-addons" />
```

Comment:
done .. Travis compiles successfully now. 

Also I did the changes you asked for